### ISSUE TITLE
Some cray-cray login bizniz happinin' here.

### ISSUE THREAD

**Issue by dtgay** [fedora-infra/tahrir#85]
1. I discover @decause can't login.
2. @decause wonders aloud if it's because he's not in the admin group. I dismiss this as ridiculous because his Login link didn't change to Logout, and that is based on the `logged_in` variable, not on administrator privs.
3. I explore other possibilities. I am able to login on his laptop with my account. His account continues to cause the Tahrir application to act as if no one is logged in. Nothing strange in server log.
4. I see that he isn't in the admins list on the cloud node (I thought he was, but regardless, like I said, that shouldn't matter). That _can't_ be the problem.
5. I add him to the admins list.
6. He can now login.
7. I removed him from the list and asked him to try again, to confirm that it was indeed the admin list causing the issue.
8. He no longer can log in.

**tl;dr apparently, not in the tahrir.admin list, not loggin' in**

:(
